### ISSUE TITLE
(Bug): See Bug 424 - Extension not working anymore?

### ISSUE THREAD

**Issue by Pentazer** [Anarios/return-youtube-dislike#428]
### Browser

Brave

### Browser Version

Version 1.34.80 Chromium: 97.0.4692.71 (Official Build) (64-bit)

### Extension or Userscript?

Extension

### Extension/Userscript Version

v2.1.0.3

### Video link where you see the problem

https://www.youtube.com/watch?v=8L7v3vRA65E

### What happened?

[A bug happened!](https://github.com/Anarios/return-youtube-dislike/issues/424)

![Screenshot 2022-01-10 064526](https://user-images.githubusercontent.com/54109677/148777742-dae32200-d3cd-4a93-b016-625bef85e1d6.jpg)

This bug is persisting and has not been resolved. I've disabled, re-enabled, uninstalled and reinstalled the extension this morning. Still not working.

**Comment by Pentazer** [comment]
![Screenshot 2022-01-10 070057](https://user-images.githubusercontent.com/54109677/148778351-0ebe3f3d-4d8b-4225-81d3-84c0dc1806be.jpg)
See this 2018 video for reference, still shows only the dislike button, but no results.

**Comment by Anarios** [comment]
what do you see when you open https://returnyoutubedislikeapi.com/votes?videoId=QOFEgexls14 ?

Which other extensions do you have?

**Comment by Pentazer** [comment]
That link returns:
{"id":"QOFEgexls14","dateCreated":"2022-01-10T14:08:12.842569Z","likes":3081,"dislikes":226,"rating":4.716440422322775,"viewCount":30914,"deleted":false}

Which I think is the same as describe din the troubleshooting steps. But I just discovered this:

For some reason when those changes were happening that caused this whole mess in the first place it turns out the script that drives this extension suddenly became present in my script blocker extension (to note: this was not the case prior to this update, it worked as is). When you mentioned other extensions I checked the script blocker and to my surprise, I saw that youtubedislike is now a newly present script that was not there before. Allowing the script restores functionality. The thing that strikes me as odd is that link you provided works and returns the correct information and prior to this update, I didn't need to manually allow the extension, only now is that coming up. 

Regardless, I've allowed the script and now it works! So whatever the update was, that would be something to add to the troubleshooting steps as that seems to be new.
